packages/mesos/README.md

```diff
@@ -2,7 +2,7 @@
 
 These commits can be found in the repository at <a href="https://github.com/mesosphere/mesos/">https://github.com/mesosphere/mesos/</a>:
 
-<li>[20e5654a294f599b637efadf32206e7b15398081] Set LIBPROCESS_IP into docker container.
-<li>[098743933a99925cfccb19f132393fa9f236c31e] Mesos UI: Change paths, pailer, and logs to work with a reverse proxy.
-<li>[f01e1ecc7ee9e9879d7059f8862a389f9f9c25c6] Revert "Fixed the broken metrics information of master in WebUI."
-<li>[17979fea307b7c094c2bac8e5a3f25af33208a0c] Updated mesos containerizer to ignore GPU isolator creation failure.
+<li>[461a5515c0994b87e5debe85f073c5eabf769fd4] Set LIBPROCESS_IP into docker container.
+<li>[8e7df46a9b0ff8f096a975db4edf66a2d38fa408] Mesos UI: Change paths, pailer, and logs to work with a reverse proxy.
+<li>[ede729a5d008dcebaca87cf564fca63185d20fd4] Revert "Fixed the broken metrics information of master in WebUI."
+<li>[4fd55cefc70fb7e42def437cec594bebb8ed90a7] Updated mesos containerizer to ignore GPU isolator creation failure.
```

Comment:
Were there any merge conflicts when cherry-picking?
If not, it seems unnecessary/excessive to update these SHAs for every Mesos bump.